modules/ingestor/src/main/scala/package.scala

```diff
@@ -32,8 +32,7 @@ extension [A](changes: List[ChangeStreamDocument[A]])
   def unique: List[ChangeStreamDocument[A]] =
     changes
       .foldRight(List.empty[ChangeStreamDocument[A]] -> Set.empty) { case (change, p @ (acc, ids)) =>
-        val id = change.docId.getOrElse("")
-        if !ids.contains(id) && id != ""
+        if change.docId.exists(ids.contains)
```

Comment:
hi @lenguyenthanh this should actually be `if change.docId.exists(!ids.contains)`

Reply:
pushed the patch here - https://github.com/gravetii/lila-search/pull/2

Reply:
pushed the patch here - https://github.com/lichess-org/lila-search/pull/308